compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -1891,7 +1891,7 @@ fn add_order_independent_options(
     cmd.optimize();
 
     // Pass debuginfo and strip flags down to the linker.
-    cmd.debuginfo(strip_value(sess));
+    cmd.debuginfo(strip_value(sess), &codegen_results.crate_info);
```

Comment:
Can you pass just `used_crates_natvis` to `debuginfo()`?

Reply:
Also maybe combine ``crate_info.used_crates_natvis` and `sess.opts.cg.natvis` in advance?